app/assets/javascripts/initializers/initializeTimeFixer.js

```diff
@@ -7,26 +11,26 @@ function initializeTimeFixer() {
     return;
   }
 
-  function convertUtcTime(utcTime) {
+  function convertUtcTime(utcT) {
```

Comment:
I think a better refactoring would be to move these functions outside the containing function if possible

Reply:
Yeah, sounds good